src/index.js

```diff
@@ -323,7 +323,15 @@ class Dropzone extends React.Component {
       type: 'file',
       style: { display: 'none' },
       multiple: supportMultiple && multiple,
-      ref: el => this.fileInputEl = el, // eslint-disable-line
+      ref: el => {
```

Comment:
Can you please extract this to a instance method to prevent from creating a new function each time?